bin/Powheg/lheMacros/mergeLheFiles.cpp

```diff
@@ -44,6 +44,7 @@ int main(int argc, char** argv)
       eventIt++;
     }	
 
+<<<<<<< HEAD
```

Comment:
this conflict is related to the change from 
</weightgroup>
to
</initrwgt>
I guess is due to the reweighting scheme adopted (i.e. reweight the entire LHE file or event by event) but I am not fully sure what to do